static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -54,6 +54,41 @@
     position: relative;
 }
 
+.youtube-media-atom__paid-for-overlay {
+    background-color: #ffffff;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    height: 85px;
+    opacity: .5;
+}
+
+.youtube-media-atom__paid-for-label {
+    font-size: 16px;
+    color: #000000;
+    margin: 5px 10px 0 0;
+    @include mq($until: 590px) {
```

Comment:
What does `590px` relate to? Are none of the `$mq-breakpoints` defined in `static/src/stylesheets/_vars.scss`?

Reply:
I have checked the breakpoints but the label was overlapping with the logo between `mobileLandscape` and `phablet`. Though to be more precise using fixed pixels but could switch to either: 
- closest smaller breakpoint `mobileLandscape` -> That would mean if user  resizes on the browser screen somewhere between those two they could see the overlapping text
- closest bigger breakpoint  `phablet` -> We won't have any overlapping issue, but there would mean we don't use some space to show the label for few pixels. Potentially very rare case?